### DISCUSSION TITLE
Exclude internal wall short paths

### DISCUSSION THREAD

**Discussion by VitaliiSadovshikov** [OrcaSlicer/OrcaSlicer#4692] [Q&A] [answered]
In the figure below, two short paths for the internal wall are shown that I would like to exclude from printing. The problem with these short paths is that the filament extruded on these paths often creates obstacles for the nozzle on upcoming paths in the layer, meaning the nozzle collides with the filament of those paths. The question is, is it possible to eliminate short paths from printing, for example, by considering the length of the paths?

![Image](https://github.com/SoftFever/OrcaSlicer/assets/92667893/b3932c7b-ca39-4344-a2df-8afcac038c60)

This is a .3mf file (the image above shows the fourth layer):
[Discourse.ExcludeShortPaths.3mf.zip](https://github.com/SoftFever/OrcaSlicer/files/14741192/Discourse.ExcludeShortPaths.3mf.zip)

**Comment by igiannakas** [comment, marked as answer]
You can enable the minimum wall length parameter under the wall generator group - set it to 2.0 to filter these extrusions out.

**Reply by VitaliiSadovshikov** [reply]
Very grateful, this really solves the problem! If you have a couple of minutes, could you tell me what this value means? Is it mm or some factor since the short segments are about 0.4mm long and to exclude them I have to set the value to more than 2

**Reply by igiannakas** [reply]
It’s a factor against the nozzle width I believe.
